.gitignore

```diff
@@ -79,5 +79,4 @@ client-secret.json
 e2e-tests/conf/cloud-secret.yml
 .DS_Store
 .idea/**
-e2e-tests/init-deploy/smallrun
-versionserviceclient/
+e2e-tests/init-deploy/smallrun
```

Comment:
It tells you need one more line at the end of the file

Reply:
5cb5cd1